### PR TITLE
Pd 450 cli guide document system config namespace

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALECLIReference/System/CLIConfig.md [hunk 1]
```diff
@@ -12,6 +12,111 @@ tags:
 {{< toc >}}
 
 {{< include file="/_includes/CLIGuideWIP.md" >}}
+{{< include file="/_includes/SCALECLIIntroduction.md" >}}
+
+## Config Namespace
+The **config** namespace has 3 commands and is based on configuration management functions found in the SCALE API and web UI.
```
Comment (micjohnson777) [inline content/SCALECLIReference/System/CLIConfig.md:18]: numbers in the 1-10 range are spelled out.
```suggestion
The **config** namespace has three commands and is based on configuration management functions found in the SCALE API and web UI.
```

File: content/SCALECLIReference/System/CLIConfig.md [hunk 2]
```diff
@@ -12,6 +12,111 @@ tags:
...
+Enter the command string then press <kbd>Enter</kbd>.
+After entering the command correctly, the job begins and progress displays.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+`system config reset`
+
+Press <kbd>Enter</kbd>.
```
Comment (micjohnson777) [inline content/SCALECLIReference/System/CLIConfig.md:47]: this does not belong here. delete this line.
```suggestion
```

File: content/SCALECLIReference/System/CLIConfig.md [hunk 3]
```diff
@@ -12,6 +12,111 @@ tags:
...
+{{< /expand >}}
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>system config save > <i>/mnt/tank/test/FILENAME.tar</i></code>
+
+Where */mnt/tank/test/FILENAME.tar* is the desired file name and save path.
+Press <kbd>Enter</kbd>
```
Comment (micjohnson777) [inline content/SCALECLIReference/System/CLIConfig.md:103]:
```suggestion
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Just a few things. I didn't do the edits because I saw this in the other PR I edited so I'm just doing a review this time.

**Comment by DjP-iX** [comment]
Converting to draft to revisit and add warnings to the `reset` command per @kmoore134's suggestion

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**